Added information about comments sorting (#29)
minimaxir/hacker-news-undocumented: README.md

HN submissions which are tutorials are [downranked by moderators](https://news.ycombinator.com/item?id=16485753), as they gratify intellectual curiosity less.

### Comments sorting

> Comments are sorted by their not visible scores. Some additional information can be found [here](https://news.ycombinator.com/item?id=7287535).

Reviewer: ```suggestion
Comments are sorted by their invisible scores. Some additional information can be found in [this comment](https://news.ycombinator.com/item?id=7287535).
```
